README.md

```diff
@@ -55,7 +55,33 @@ Run `xattr -d com.apple.quarantine path-to-whisky`
 
 There is an issue where the libraries for font rendering are not properly bundled yet. You will need to install freetype.
 
-- Open an x86 terminal session with `arch -x86_64 zsh`
-- Install brew `/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"` (If you run into issues, let me know)
+
+### Setup your development and Homebrew environment
+- Ensure the Command Line Tools for Xcode 15 beta are installed. Visit https://developer.apple.com/downloads to download these tools.
+- Enter an x86_64 shell to continue the following steps in a Rosetta environment. All subsequent commands should be run within this shell.
+arch -x86_64 zsh
+
+- Install the x86_64 version of Homebrew if you don't already have it.
+`/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"`
+
 - Install freetype `/usr/local/bin/brew install freetype`
-- Restart Whisky and remake the bottle
+
+- The Game Porting Toolkit runs under Rosetta 2. Ensure that Rosetta 2 is installed.
+`softwareupdate --install-rosetta`
+
+- Make sure the brew command is on your path:
+ `which brew`
+If this command does not print `/usr/local/bin/brew`, you must either modify your PATH to put `/usr/local/bin` first, or fully specify the path to brew in the subsequent commands.
```

Comment:
Can be reduced to a step that just tells the user what to do here unconditionally (e.g., Execute: `export PATH=...`).

Reply:
wouldn't this conflict with the native brew arm64 brew installed in /opt/homebrew/bin/brew?